src/chrome/content/rules/CNET.com.xml

```diff
@@ -0,0 +1,47 @@
+<!--
+	Non-functional hosts
+		Couldn't connect to server:
+		- blogs.cnet.com
+		- export.cnet.com
+
+		Timeout was reached:
+		- ces.cnet.com
+		- china.cnet.com
+		- cnettv.cnet.com
+		- crave.cnet.com
+		- forums.cnet.com
+		- news.cnet.com
+		- reviews.cnet.com
+		- shopper.cnet.com
+
+		SSL peer certificate was not OK:
+		- descargar.cnet.com
+		- live.cnet.com
+		- paidcontent.cnet.com
+		- telecharger.cnet.com
+
+		Different content:
+		- asia.cnet.com
+
+-->
+<ruleset name="CNET.com">
+	<target host="cnet.com" />
```

Comment:
`www.cnet.com`

Reply:
www is currently preloaded, see https://hstspreload.org/?domain=www.cnet.com